docs/faqs/faqs.md

```diff
@@ -0,0 +1,212 @@
+
+  
+</details>
+
+<details>
+  <summary>Is Amplication Free?</summary>
+  
+ The Community Edition (CE) is free and always will be. However, to ensure that we can continue to develop Amplication, the Enterprise Edition (EE) will provide enterprise functionality that may require payment. For more information about the CE and EE, read:
+
+ 
+ [Amplication’s licensing model](https://docs.amplication.com/docs/about/about/licensing/)
+
+  
+</details>
+
+<details>
+  <summary>What is the difference between the Community Edition and the Enterprise Edition?</summary>
+
+  
+  **Community Edition (CE)** – contains Amplication's core functionality. It is open-source and free to use for the whole community. The Amplication Community Edition continues to be licensed under the [Apache 2.0 license](https://github.com/amplication/amplication/blob/master/LICENSE).
+
+  **Enterprise Edition (EE)** - adds advanced operations, security, and commercially orientated features. The Enterprise Edition is built on top of the Community Edition and the additional functionality will be supplied as source-available. While users can view and download the code, ownership of the code remains with Amplication, meaning users cannot resell it. Use of the Amplication Enterprise Edition is subject to the Amplication [Terms of Service](https://amplication.com/terms)
+
+  
+</details>
+
+
+<details>
+  <summary>Who owns the code I generate with Amplication?</summary>
+  
+You own the code.  Develop your app however you wish, and deploy it wherever want; on public cloud, private cloud, or on-premise. 
+  
+</details>
+
+
+<details>
+  <summary>How can I get the code?</summary>
+  
+You can download the source code of your application or push a new Pull Request to a GitHub repository. There is also a code-view option, that enables you to preview the code before it is generated. 
+  
+  
+</details>
+
+
+<details>
+  <summary>What server-side technologies does Amplication use?</summary>
+  
+NestJS, Prisma, PostgreSQL, Passport, GraphQL, Swagger UI, Jest, Docker
+  
+</details>
+
+
+<details>
+  <summary>What client-side technologies does Amplication use?</summary>
+  
+ReactJS, React-Admin, Axios, Formik 
+  
+</details>
+
+
+<details>
+  <summary>Is Amplication a no-code solution?</summary>
+  
+Amplication is a tool for professional developers, enabling you to automate repetitive tasks, eliminating or reducing the time spent on repetitive server-side tasks and boilerplate code. However, your developer skills are very much required to create your  hand-crafted business logic, so it is definitely NOT a no-code solution. Some would call Amplication a low-code solution because it reduces the amount of code you need to write. 
+  
+</details>
+
+
+<details>
+  <summary>What is the Amplication Architecture?</summary>
+  
+The generated app is built from two projects, each in a separate folder:
+
+- Server - For all the server components including REST API, GraphQL, Services and more.
```

Comment:
- Server - for all the server components including REST API, GraphQL, Services and more.